tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -1666,4 +1666,62 @@ public function userTriesToListThePermissionsOfSpaceOwnedByUsingRootEndpointOfTh
 	public function userRemovesTheLastLinkShareOfSpaceUsingPermissionsEndpointOfGraphApi(string $user, string $space):void {
 		$this->featureContext->setResponse($this->removeAccessToSpaceItem($user, 'link', $space, ''));
 	}
+
+	/**
+	 * @param string $share
+	 * @param string $sharee
+	 * @param string $sharer
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 * @throws Exception
+	 */
+	public function checkIfShareExists(string $share, string $sharee, string $sharer): void {
+		// check share mountpoint
+		$response = GraphHelper::getMySpaces(
+			$this->featureContext->getBaseUrl(),
+			$sharee,
+			$this->featureContext->getPasswordForUser($sharee),
+			"",
+			$this->featureContext->getStepLineRef()
+		);
+		$driveList = HttpRequestHelper::getJsonDecodedResponseBodyContent($response)->value;
+		$foundShareMountpoint = false;
+		foreach ($driveList as $drive) {
+			if ($drive->driveType === "mountpoint" && $drive->name === $share && $drive->root->remoteItem->driveAlias === "personal/brian") {
```

Comment:
Why do we hardcode "brian" here?
IMO you can lowercase `$sharer` and append it to `personal/`